.github/stale.yml

```diff
@@ -4,8 +4,9 @@ daysUntilStale: 90
 daysUntilClose: 7
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - pinned
-  - security
```

Comment:
I've just updated the 4 issues that were `pinned` so that they are either `kind/bug` or `kind/feature`.